### PR TITLE
Update readers for stgc and fcs

### PR DESCRIPTION
**Pull request by jml985**
update the offline readers according to the code running online.

### CODE REVIEW

File: OnlTools/Jevp/StJevpBuilders/daqBuilder.h [hunk 1]
```diff
@@ -6,8 +6,15 @@ class daqReader;
 #include <TH1F.h>
 #include <TH2F.h>
 
+// Git update test from daqman
+// and add to that too
+// again...
```
Comment (plexoos) [inline OnlTools/Jevp/StJevpBuilders/daqBuilder.h]: Please remove or confirm that you intend to commit these lines

File: OnlTools/Jevp/StJevpBuilders/daqBuilder.h [hunk 2]
```diff
@@ -6,8 +6,15 @@ class daqReader;
...
 #include <math.h>
 
+// GIT update test from rts02
+//
+// asdf
+//
```
Comment (plexoos) [inline OnlTools/Jevp/StJevpBuilders/daqBuilder.h]: Please remove or confirm that you intend to commit these lines

File: StRoot/RTS/src/TONKO_TRASH/yada.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
+1) Tonko: line XX
+2) Tonko: line 2
+
+
+3)  Jeff: added this and Tonko modified the line
+
```
Comment (plexoos) [inline StRoot/RTS/src/TONKO_TRASH/yada.txt]: Please remove or confirm that you intend to commit this file

File: StRoot/RTS/include/FTP/ftpc_padkey.h [hunk 4]
```diff
@@ -0,0 +1,19378 @@
+#ifndef _FTP_PADKEY_H_
```
Comment (plexoos) [inline StRoot/RTS/include/FTP/ftpc_padkey.h:1]: Is this file really used by the stgc and fcs readers?

Comment (jml985) [reply StRoot/RTS/include/FTP/ftpc_padkey.h:1]: No, but it was left out when the offline github was created, I guess.   It is still accessed by the online code, and may be needed for historical readers.  I'm not sure exactly what its needed for but please include it in the directory structure.

**Comment by plexoos** [comment]
Is it ready to be merged?

**Comment by jml985** [comment]
Yes, as far as I am concerned.   It appears to still need another reviewer according to github though, and also I'm not sure who does the actual merge.   Do I press squash and merge when the reviews are all in, or does the last reviewer do it, or you?

**Comment by plexoos** [comment]
I can merge if you say there is no need to wait for responses from the other maintainers designated in .github/CODEOWNERS
I hope it is ok with them to give up their responsibilities. Or you can wait for two approvals and press the green button to squash and merge. Any user with push access can do that. 

This PR should be definitely squashed into a single commit in order not to contaminate the main branch with pointless commits and log messages such as "test git" and "update".

**Comment by jml985** [comment]
I'd go ahead and do the merge.   This is after all the code that is ***already*** running online.   The PR was from me but the code changes were from Tonko and Akio, so if there were to be a reviewer it should be me :-)

File: StRoot/RTS/include/FTP/ftpc_padkey.h (hunk 4 above)
Comment (plexoos) [reply StRoot/RTS/include/FTP/ftpc_padkey.h:1]: This file conflicts with StRoot/RTS/include/FTP/FTPC_PADKEY.h on case-insensitive systems such as macOS. If the lower-case version `ftpc_padkey.h` is used by the online code, can we remove or rename the upper-case one `FTPC_PADKEY.h`?

**Pull request merged**